code/_helpers/items.dm

```diff
@@ -0,0 +1,9 @@
+//Prevents robots dropping their modules.
+/proc/dropsafety(var/atom/movable/A)
+	if (istype(A.loc, /mob/living/silicon))
```

Comment:
https://puu.sh/tiZdI/7100b9b26e.png

Just uh. Delete the helper in unsorted.dm. It's not used anywhere.